src/store/metadata/stub/content/datasets.json

```diff
@@ -267,8 +267,53 @@
               "modified": "2022-08-18T10:51:00Z"
             }
           ]
+      },
+      {
+        "@id": "https://staging.idpd.uk/datasets/DLUHC",
```

Comment:
Needs a globally unique identifier, something like `epcs-for-existing-domestic-properties`

Reply:
set globally unique identifier as epcs-for-existing-domestic-properties